src/conformance/conformance_test/test_LayerComposition.cpp

```diff
@@ -349,21 +350,25 @@ namespace Conformance
                 const float t = y / 255.0f;
                 const XrColor4f dst = Colors::Green;
                 const XrColor4f src{0, 0, t, t};
-                const XrColor4f blended{dst.r * (1 - src.a) + src.r, dst.g * (1 - src.a) + src.g, dst.b * (1 - src.a) + src.b, 1};
+                const XrColor4f blended{dst.r * (1 - src.a) + src.r, dst.g * (1 - src.a) + src.g, dst.b * (1 - src.a) + src.b, 0};
                 blueGradientOverGreen.DrawRect(0, y, blueGradientOverGreen.width, 1, blended);
             }
 
             const XrSwapchain answerSwapchain = compositionHelper.CreateStaticSwapchainImage(blueGradientOverGreen);
-            interactiveLayerManager.AddLayer(
-                compositionHelper.CreateQuadLayer(answerSwapchain, viewSpace, 1.0f, XrPosef{Quat::Identity, {0, 0, QuadZ}}));
+            XrCompositionLayerQuad* truthQuad =
+                compositionHelper.CreateQuadLayer(answerSwapchain, viewSpace, 1.0f, XrPosef{Quat::Identity, {0, 0, QuadZ}});
+            truthQuad->layerFlags |= XR_COMPOSITION_LAYER_UNPREMULTIPLIED_ALPHA_BIT;
```

Comment:
(copying over from the older PR)
why is the unpremultipled alpha flag set? it shouldn't do anything when blending isn't enabled

Reply:
> (copying over from the older PR)
> why is the unpremultipled alpha flag set? it shouldn't do anything when blending isn't enabled

My intention was to keep these as two distinct PRs, so we could talk about this over there.

It should have no effect, but if you were a runtime that support one flag but not the other, the unpremultiplied bit made it more obvious that you were reading the texture alpha when you shouldn't be. Otherwise you'll blend green or the demo image on top of the black background: new_dst = ( 1 - alpha) * dst + src

dst is black, src is programmer green, so alpha doesn't matter.  If you use non-premultiplied alpha you get this:
new_dst = (1 - alpha)* dst + alpha*src

With the same source texture, that results in black instead of bright green.

Reply:
Ok that makes sense. A short comment would be useful so no one thinks its a mistake or something (and it might help someone debugging this kind of bug spot the underlying problem)

Reply:
Why have a small comment when  large one will do?

Reply:
SteamVR was such a runtime on my machine, and the failure was invisible. So I added the flag to make it obvious.